Library/Homebrew/requirements/linux_requirement.rb

```diff
@@ -0,0 +1,9 @@
+class LinuxRequirement < Requirement
+  fatal true
+
+  satisfy(build_env: false) { OS.linux? }
```

Comment:
@sjackman: do we want to add a check for Linux kernel version?

Reply:
Currently the only formula that has a minimum Linux kernel version requirement is `glibc`. So I don't feel there's a need for this feature. If ≥ 3 formulae require  a minimum version of the Linux kernel, then yes.